tcks/microprofile-opentracing/base/src/main/java/io/quarkus/tck/opentracing/DeploymentProcessor.java

```diff
@@ -0,0 +1,36 @@
+package io.quarkus.tck.opentracing;
+
+import org.eclipse.microprofile.opentracing.ClientTracingRegistrarProvider;
+import org.jboss.arquillian.container.test.spi.client.deployment.ApplicationArchiveProcessor;
+import org.jboss.arquillian.test.spi.TestClass;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+
+import sun.security.jca.Providers;
```

Comment:
Wrong import, need the JAX-RS one

Reply:
Haha, good catch. This proves that we don't need this at all and the `ExceptionMapper` is detected automatically.